test/services/ReviewServiceSpec.scala

```diff
@@ -287,13 +333,14 @@ class ReviewServiceSpec extends UnitSpec with MockFactory with ReviewData with A
   "Calling the approvalPageInfo method" when {
 
     trait PageInfoTest extends Test {
+      val currentDateTime: LocalDateTime = LocalDateTime.now()
       val processReview: ApprovalProcessReview = ApprovalProcessReview(
         UUID.randomUUID(),
         "validId",
         1,
         ReviewType2i,
         "This is the title",
-        List(ApprovalProcessPageReview("1", "/pageUrl", Some("result1")), ApprovalProcessPageReview("2", "/pageUrl2", Some("NotStarted")))
+        List(ApprovalProcessPageReview("1", "/pageUrl", Some("result1")), ApprovalProcessPageReview("2", "/pageUrl2", Some("NotStarted"), updateDate = currentDateTime))
```

Comment:
You could define the ApprovalProcessPageReview instances above to prevent the very long lines.

Reply:
I'll reformat